mobile/Features/Journal/Screens/JournalPage.jsx

```diff
@@ -39,63 +102,129 @@ export default function JournalPage({ navigation }) {
+              <SafeAreaView style={styles.textBox}>
                 <ScrollView automaticallyAdjustKeyboardInsets>
-                  <TextInput multiline placeholder="Type your response" onChangeText={setText} value={text} />
-                  <View style={{ height: 40 }} />
-                </ScrollView>
-              </View>
-              <Button title="Submit" onPress={handlePopUp} />
-              <Modal visible={confirmPopUp}>
-                <TouchableOpacity onPressOut={handlePopUp} style={styles.modalView}>
-                  <View style={styles.modalBox}>
-                    <Text style={{ fontSize: 20 }}>confirm journal entry?</Text>
-                    <Pressable
-                      style={styles.modalSelections}
-                      onPress={() => addNewJournal(username, prompt, text)}
-                    >
-                      <Text>
-                        yes
-                      </Text>
-                    </Pressable>
-                    <Pressable style={styles.modalSelections} onPress={handlePopUp}>
-                      <Text>
-                        no
-                      </Text>
-                    </Pressable>
+                  <View>
+                    <TextInput multiline placeholder="Type your response" onChangeText={setText} value={text} />
                   </View>
-                </TouchableOpacity>
-              </Modal>
-            </View>
-          </View>
+                </ScrollView>
+              </SafeAreaView>
+              <Text>
+                word count:
+                {' '}
+                {wordsLen(text)}
+              </Text>
+              <SafeAreaView>
+                <Modal visible={confirmPopUp}>
+                  <TouchableOpacity onPressOut={handlePopUp} style={styles.modalView}>
+                    <SafeAreaView style={styles.modalBox}>
+                      <Text style={{ fontSize: 20 }}>confirm journal entry?</Text>
+                      <Pressable
+                        style={styles.modalSelections}
+                        onPress={() => addNewJournal(username, title, text)}
+                      >
+                        <Text>
+                          yes
+                        </Text>
+                      </Pressable>
+                      <Pressable style={styles.modalSelections} onPress={handlePopUp}>
+                        <Text>
+                          no
+                        </Text>
+                      </Pressable>
+                    </SafeAreaView>
+                  </TouchableOpacity>
+                </Modal>
+              </SafeAreaView>
+
+            </SafeAreaView>
+          </SafeAreaView>
         </TouchableWithoutFeedback>
+        <Text>{' '}</Text>
+        <Text>{' '}</Text>
+        <Text>{' '}</Text>
+        <Text>{' '}</Text>
+        <Text>{' '}</Text>
+        <Text>{' '}</Text>
+        <Text>{' '}</Text>
+        <Text>{' '}</Text>
```

Comment:
so sorry could you explain this lol? is this the workaround i heard about?

Reply:
got rid of this, scrollview bug fixed for now (if height of textbox is set to a numerical value like 200, it works but if it's set to a percentage of the screen i.e. 70%, it doesn't work)